bot/seasons/pride/pride_anthem.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from json import load
+from pathlib import Path
+from random import choice
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class PrideAnthem(commands.Cog):
+    """Embed a random youtube video for a gay anthem!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.anthems = self.load_vids()
+
+    def get_video(self, genre: str = None) -> dict:
+        """
+        Picks a random anthem from the list.
+
+        If `genre` is supplied, it will pick from videos attributed with that genre.
+        If none can be found, it will log this as well as provide that information to the user.
+        """
+        if not genre:
+            return choice(self.anthems)
+        else:
+            songs = [song for song in self.anthems if genre.casefold() in song['genre']]
+            try:
+                return choice(songs)
+            except IndexError:
+                log.info('No videos for that genre.')
+
+    @staticmethod
+    def load_vids() -> list:
+        """Loads a list of videos from the resources folder as dictionaries"""
+        with open(Path('bot', 'resources', 'pride', 'anthems.json').absolute(), 'r') as f:
```

Comment:
A lot of other cogs across seasonalbot use the previous method of using `pathlib.Path` - surely it'd be best to keep it as it was so it's the same across the board?

Reply:
That's why I had done that originally, it's the style I saw. But I prefer the second way myself so I wasn't going to argue.

Reply:
The former is indeed the prevalent style that's been used almost everywhere. I'd actually suggest reverting that change back to the original version to keep the code style consistent.

Reply:
just because other parts of the bot are messy doesn't mean we shouldn't correct course.

I'd have no problem if @AvianAnalyst wanted to push a so-called kaizen commit to this PR where she changed all other instances of this across seasonalbot to use this second method.

Reply:
I'm on my way to work right now but I would be happy to when I get home.

Reply:
Sure, we could do that, but this pattern has been used and reviewed a lot in this repository. That could just be because everyone is copying the existing style, but the number of reviews that went over it makes me suspect that there actually was a preference for it. 

I'll try to dig up if there have been discussions on this before.

Reply:
I wasn't expecting this to be controversial. I doubt there's been much discussion about it, it's a perfectly valid way to do it, it just happens to be hideous. but hey, if I'm the only one who thinks so I'll back off.

Reply:
I don't think it's that controversial, I agree with your stylistic input, as does AvianAnalyst, so I'm not opposed to changing it. I'm just saying that this style is everywhere, so I suspected that there were some people strongly in favor of it.

Reply:
I suppose I'll chime in and say I am with you @lemonsaurus. I don't think I've ever seen only _literal_ strings being joined with pathlib since pathlib smart enough to take figure out the separators its given anyway.

Reply:
@AvianAnalyst we've discussed it internally and we all agree that the former method is really ugly compared to the latter, and that it would be great if you could do a kaizen commit changing all the ones you can find in the project.

it's fine that this just goes in this PR, even though it's out of scope.